Tests/Marketplace/zip_packs.py

```diff
@@ -212,12 +239,13 @@ def main():
     storage_bucket_name = option.bucket_name
     zip_path = option.zip_path
     artifacts_path = option.artifacts_path
-    service_account = option.service_account
-    circle_build = option.circle_build
-    branch_name = option.branch_name
-    gcp_path = option.gcp_path
+    storage_base_path = option.storage_base_path
     remove_test_playbooks = option.remove_test_playbooks
     private_build = option.private
+
+    if storage_base_path:
+        GCPConfig.STORAGE_BASE_PATH = storage_base_path
```

Comment:
`STORAGE_BASE_PATH` is both an attribute of an import as well as being a constant, as indicated by it being capitalized. Please do _not_ assign to this variable.

Reply:
I actually copied this behaviour, as suggested by @guyfreund, from copy_and_upload_packs.py file.

Do you have another solution? Can I use this practice as it is already being used in other files?
 
<img width="675" alt="Screen Shot 2021-06-15 at 14 23 29" src="https://user-images.githubusercontent.com/79846533/122044544-4db75f00-cde5-11eb-9ee8-bb9bc2251957.png">

Reply:
This is bad practice and I would avoid doing this if there is a viable alternative.